### DISCUSSION TITLE
Track loading being deprecated in V4

### DISCUSSION THREAD

**Discussion by angelobreuer** [lavalink-devs/Lavalink#823] [Q&A] [answered]
While preparing the migration for Lavalink V4 I stumbled upon the documentation indicating track loading being deprecated in V4.

I know that we can load tracks more efficiently by using the PATCH player RESTful HTTP endpoint.

Some users like to use queuing features for their music bots. It would be possible to implement this by only queuing the query and resolving it lazily later. However, we would lose functionality to implement showing previews of the track enqueued (as we do not know what the track is, before we play it).

Will there be an alternativem or something similar in V4?

---

ref: https://github.com/freyacodes/Lavalink/blob/dev/IMPLEMENTATION.md#track-loading

**Comment by topi314** [comment, marked as answer]
`/loadtracks?identifier=dQw4w9WgXcQ` is getting removed in `v4`
`/v3/loadtracks?identifier=dQw4w9WgXcQ` is there to stay and `/v4/loadtracks?identifier=dQw4w9WgXcQ` is getting added

You can already check the [`IMPLEMENTATION.md`](https://github.com/freyacodes/Lavalink/blob/v4/IMPLEMENTATION.md) on the `v4` branch. There might be some minor stuff missing

**Comment by angelobreuer** [comment]
Thank you for the quick response! I did not look at the v4 branch

**Reply by topi314** [reply]
the `v4` branch should already be usable. Feel free to report issues in the pr as you stumble across them. https://github.com/freyacodes/Lavalink/pull/818
